src/pcapi/core/users/external/sendinblue.py

```diff
@@ -137,9 +137,17 @@ def make_update_request(payload: UpdateSendinblueContactRequest) -> bool:
 
     except SendinblueApiException as exception:
         if exception.status == 524:
-            logger.warning("Timeout when calling ContactsApi->create_contact: %s\n", exception)
+            logger.warning(
+                "Timeout when calling ContactsApi->create_contact: %s\n",
```

Comment:
On devrait pouvoir supprimer le saut de ligne dans le message loggé :

```suggestion
                "Timeout when calling ContactsApi->create_contact: %s",
```